openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -52,8 +52,9 @@ start_service() {
     monitored_interfaces="--monitored_interfaces $monitored_interfaces"
 
     procd_open_instance "openwisp_monitoring_monitoring"
-    procd_set_param command $PROG "$base_url" "$uuid" "$key" "$verify_ssl" "$interval" "$monitored_interfaces"
+    procd_set_param command $PROG $base_url $uuid $key $verify_ssl $interval $monitored_interfaces
```

Comment:
Reverting quotes as this was causing `monitoring.agent` to read `--url http://www.....` as one word